projects/odf_transform/odf_transform/odf_parser/odf_parser.py

```diff
@@ -321,14 +439,14 @@ def parse_odf_code_variable(var_name):
     the index (01-99) and generate a standardized name with two digit index values if available.
     Some historical data do not follow the same standard, this tool tries to handle the issues found.
     """
-    var_list = var_name.rsplit('_', 1)
-    var_dict = {'name': var_list[0]}
-    var_dict['standardized_name'] = var_dict['name']
-    if len(var_list) > 1 and var_list[1] not in ['']:
-        var_dict['index'] = int(var_list[1])
-        var_dict['standardized_name'] += '_{0:02.0f}'.format(var_dict['index'])
-    elif len(var_list) > 1 and var_list[1] == '':
-        var_dict['standardized_name'] += '_'
+    var_list = var_name.rsplit("_", 1)
+    var_dict = {"name": var_list[0]}
+    var_dict["standardized_name"] = var_dict["name"]
+    if len(var_list) > 1 and var_list[1] not in [""]:
```

Comment:
if you're checking if var_list[1] isnt empty you could do
`if len(var_list) > 1 and var_list[1]:`